### ISSUE TITLE
immutant.org domain expired

### ISSUE THREAD

**Issue by Folcon** [luminus-framework/luminus#250]
Looking at the docs, it still seems like Immutant is considered the default server, but as far as I can tell it's not currently being actively developed? The domain http://immutant.org/ appears to be down. Should the framework swap away from it being the recommended default?

**Comment by WorldsEndless** [comment]
Confirm on this; it seems that Redhat has ceased funding of Immutant and the lead developer no longer maintains it. Any further work with immutant would require a fork, particularly as current immutant is incompatible with Java 12+. 

https://www.reddit.com/r/Clojure/comments/dhx6b6/deploying_clojure_war_is_immutant_deprecated/

**Comment by yogthos** [comment]
Yeah, I switched to Jetty as the default server now, but I missed some docs it seems.
